ods_ci/tests/Tests/100__deploy/130__operators/130__rhods_operator/139__trusted_ca_bundles.robot

```diff
@@ -0,0 +1,137 @@
+*** Settings ***
+Documentation    Test Cases to verify Serverless installation
+Library    Collections
+Resource    ../../../../Resources/Page/OCPDashboard/OCPDashboard.resource
+Suite Setup    Suite Setup
+Suite Teardown    Suite Teardown
+
+
+*** Variables ***
+${RHOAI_OPERATOR_NS}    redhat-ods-operator
+${RHOAI_OPERATOR_DEPLOYMENT_NAME}    rhods-operator
+${SERVICE_MESH_OPERATOR_NS}    openshift-operators
+${SERVICE_MESH_CR_NS}    istio-system
```

Comment:
Why do we need servicemesh resources here ? Is it just the namespace to check for configmap?

Reply:
1) By default SM operator is a dependency for DSCI, which would generate an error in operator log if not present.
2) istio-system is a non reserved namespace for which the trusted ca bundle configMap would be created. 

Not using istio-system namespace, and instead creating a test namespace is another possible approach.

Reply:
Removed SM operator dependency.